README.md

```diff
@@ -79,7 +79,7 @@ The 1.0 release of ember-simple-auth introduced a lot of breaking changes, but t
 ### Basic Usage
 
 Once the library is installed, __the session service can be injected wherever
-needed in the application__. In order to e.g. display login/logout buttons
+needed in the application__. In order to display login/logout buttons
```

Comment:
@baijum Maybe it would be better written as: "For example, to display login/logout buttons..."